server/plugin/webhook.go

```diff
@@ -702,6 +702,13 @@ func (p *Plugin) postIssueCommentEvent(event *github.IssueCommentEvent) {
 		Type:   "custom_git_comment",
 	}
 
+	repoName := repo.GetFullName()
+	repoName = strings.ToLower(repoName)
+	commentID := event.GetComment().GetID()
+
+	post.AddProp("gh_repo", repoName)
+	post.AddProp("gh_object_id", commentID)
```

Comment:
This should be applied to any webhook event associated with a PR or issue, not just the "issue comment" events. For example, if a new pull request is created, I should be able to react to the post in Mattermost and have my reaction on the PR itself in GitHub. Probably best to try to de-duplicate this code addition as well.

I'm thinking we will need to add an `gh_object_type` or `gh_object_name` post prop in order know what github API call to make, e.g. `DeleteIssueCommentReaction` vs `DeleteIssueReaction`